src/ORM/FieldType/DBInt.php

```diff
@@ -64,7 +64,13 @@ public function Nice()
     {
         return sprintf('%d', $this->value);
     }
-
+    
+    public function SpelledOut()
+    {
+        $v = $this->prepValueForDB($this->value);
```

Comment:
Why are you calling `prepValueForDB()` first? e.g. `DBInt::Formatted()` doesn't do that.